server/api.js

```diff
@@ -134,4 +134,97 @@ router.get("/tenders", async (req, res) => {
 	}
 });
 
+router.get("/bid/:tenderId", async (req, res) => {
```

Comment:
`/bid/1` should refer to the bid with id 1

`/bid?tender_id=1` refers to bids associated with the tender id 1